### PR TITLE
LSR-991 Screenlets' documentation improvements (iOS)

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

File: ios/Framework/Core/Asset/ListScreenlet/AssetListScreenlet.swift [hunk 1]
```diff
@@ -13,51 +13,65 @@
...
 @objc public protocol AssetListScreenletDelegate : BaseScreenletDelegate {
 
 	/// Called when a page of assets is received. Note that this method may be called 
 	/// more than once; one call for each page received.
 	///
 	/// - Parameters:
-	///   - screenlet
-	///   - assets: list of assets.
+	///   - screenlet: Asset list screenlet instance.
+	///   - assets: List of assets.
```
Comment (nhpatt) [inline ios/Framework/Core/Asset/ListScreenlet/AssetListScreenlet.swift]: Why uppercase after : ?

File: ios/Framework/Core/FileDisplay/FileDisplayScreenlet.swift [hunk 2]
```diff
@@ -14,57 +14,77 @@
...
+/// The FileDisplayScreenletDelegate protocol defines some methods that you use to manage the
+/// FileDisplayScreenlet events. All of them are optional.
 @objc public protocol FileDisplayScreenletDelegate : BaseScreenletDelegate {
 
 	/// Called when the screenlet receives the file.
 	///
 	/// - Parameters:
-	///   - screenlet
-	///   - url: file URL.
+	///   - screenlet: Pdf display screenlet instance.
```
Comment (nhpatt) [inline ios/Framework/Core/FileDisplay/FileDisplayScreenlet.swift]: PDF, uppercase

Comment (dgarciasarai) [inline ios/Framework/Core/FileDisplay/FileDisplayScreenlet.swift]: The comment is wrong... I already changed it in develop branch 👍

File: ios/Framework/Core/Asset/ListScreenlet/AssetListScreenlet.swift (hunk 1 above)
Comment (dgarciasarai) [inline ios/Framework/Core/Asset/ListScreenlet/AssetListScreenlet.swift]: Improves readability when developers are viewing documentation in Xcode :)
quick documentation in xcode

**Pull request merged**